src/index.ts

```diff
@@ -108,6 +113,24 @@ export class Client {
     return this.request<IEmailSetResponse>('Email/set', args);
   }
 
+  public emailSubmission_get(
```

Comment:
Just to be sure you are aware of it: EmailSubmission/get is not implemented by James